src/messages/dex.ts

```diff
@@ -5,7 +5,7 @@ export default async function handleDex(m: Message) {
   const ownedPokemon = await MoreStrongPokemon.find({
     user: m.author.id,
   }).sort({ total: -1 });
-  const uniquePokemon = [...new Set(ownedPokemon.map((p) => p.id))];
```

Comment:
Don't fix the problem, with regional forms we have two names for same pokemon like `rattata` and `rattata-alola`. They are same pokemon, not two entries on dex.

Reply:
I think the best way to solve this would be to introduce something like a `dex_id` property on the Pokemon schema so the same species can have different regional names but the same pokedex entry. Would require a lot more changes though.

Reply:
OwnedPokemon has number, now you said dex_id is more efficient to communicate what is this property. I'll change this property and use to solve this issue

Reply:
Created a pr to impprove identification #12